### ISSUE TITLE
Masking function for shadow model logits

### ISSUE THREAD

**Issue by henrikfo** [aidotse/LeakPro#56]
# Feature
A function that returns a mask that indicates if audit indicies was used for training a shadow model

## Desired Behavior / Functionality
The mask can be used to extract the IN- and OUT- logits for the respective shadow model

## What Needs to Be Done
Create a function that walks through all audit indicies and shadow model training indicies and checks if the audit index is in the shadow model training samples

## How Can It Be Tested
Can be tested by using the function when excluding some indicies for shadow model trainingset.
The function should return 0 IN- samples and all OUT- samples for a offline attack
